### PR TITLE
Feature/scss

### PR DESCRIPTION
**Pull request by mdflynn**
## What is the change?
This adds variables to the scss files. Splits into more modular styling.

## What does it fix?
Removes some console.logs from test files.

## Is this a fix or a feature?
Both.

## Where should the reviewer start?
The style.scss and the variables.scss files.

## How should this be tested?
No testing currently for this pull request.

### CODE REVIEW

File: test/userrepo-test.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import User from '../src/User';
 describe('User Repo', function() {
   let user1;
   let user2;
-  let user;
```
Comment (okalimbet) [inline test/userrepo-test.js:10]: nice catch with this typo!

File: test/hydration-test.js [hunk 2]
```diff
@@ -145,7 +149,7 @@ describe('Hydration', function() {
     expect(hydration.calculateFirstWeekOunces(userRepo, 4)[6]).to.eql('2019/04/15: 36');
   });
 
-  it('should find sleep quality by day for that days week', function() {
+  it('should calculate random week ounces for a user', function() {
```
Comment (okalimbet) [inline test/hydration-test.js:152]: Thank you for rephrasing the test, now it matches the actual test which  checks hydration.

File: src/css/style.scss [hunk 3]
```diff
@@ -1,17 +1,19 @@
+@import "variables";
```
Comment (cameronRomo) [inline src/css/style.scss:1]: These variables really help to dry up the styles sheet.

File: test/hydration-test.js [hunk 4]
```diff
@@ -103,6 +103,10 @@ describe('Hydration', function() {
     hydration = new Hydration(hydrationData);
   });
 
+  it('should only take in an array', function() {
```
Comment (okalimbet) [inline test/hydration-test.js:106]: Good sad path test checking the data type. I wonder if we need to include the test to check in the array is not empty too.

**Review by okalimbet — APPROVED** [review]

File: test/hydration-test.js (hunk 4 above)
Comment (cameronRomo) [inline test/hydration-test.js:106]: Great sad path for this.

**Review by okalimbet — APPROVED** [review]
Great organization of variables! 
We might see if we can split styles by elements in html for a better organization or to make css file shorter.

**Review by cameronRomo — APPROVED** [review]
This is a great place to start with this refactor. The styles sheet is much dryer and there are some sad paths added for `hydration.js`
